simulariumio/filters/multiply_time_filter.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import copy
+import logging
+
+import numpy as np
+
+from ..data_objects import TrajectoryData, AgentData
+from .filter import Filter
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class MultiplyTimeFilter(Filter):
+    multiplier: float
+    apply_to_plots: bool
+
+    def __init__(
+        self,
+        multiplier: float,
+        apply_to_plots: bool = True,
+    ):
+        """
+        This filter multiplies time values
```

Comment:
nit: similar to above, would be helpful to specify 'scales the time-stamp values for each simulation frame by a provided multiplier'